projects/admin/src/app/circulation/main-request/requested-item/requested-item.component.html

```diff
@@ -0,0 +1,95 @@
+<!--
+ RERO ILS UI
+  Copyright (C) 2019 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<div class="row p-2 mb-1 align-middle item"
+     *ngIf="document else loading"
+     [ngClass]="getCallout()"
+  >
+
+  <div class="actions">
+    <button class="btn btn-sm btn-outline-secondary"
+            *ngIf="item.loan.state === LoanState.PENDING"
+            [attr.id]="item.barcode | idAttribute:{prefix: 'item', suffix: 'validate-button'}"
+            (click)="validateRequest()"
+    >
+      <i class="fa fa-check pr-1"></i>
+      {{ 'Validate' | translate }}
+    </button>
+  </div>
+  <!-- FIRST ROW :: item-barcode, document-title, requester-name, request-date -->
+  <div class="col-2">
+    <button type="button" class="pl-0 pt-0 btn"
+            (click)="isCollapsed = !isCollapsed"
+            [attr.aria-expanded]="!isCollapsed"
+            aria-controls="collapse">
+      <i [ngClass]="{ 'fa-caret-down': !isCollapsed, 'fa-caret-right': isCollapsed }" class="fa" aria-hidden="true"></i>
+    </button>
+    <a [routerLink]="['/records','items','detail', item.pid]">{{ item.barcode }}</a>
+  </div>
+  <div name="title" class="col-lg-5">
+    <a [routerLink]="['/records','documents','detail', document.pid]" *ngIf="document.title | mainTitle as title ">
+      {{ title }}
+    </a><br/>
+    <ng-container *ngIf="document.contribution && document.contribution.length > 0">
+      <ul class="list-inline mb-0 small">
+        <li *ngFor="let contribution of document.contribution; let last = last; let i = index" class="list-inline-item">
+          <span *ngIf="!contribution.agent.pid" [innerHTML]="contribution | contributionFormat: true"></span>
+          <a
+            *ngIf="contribution.agent.pid"
+            [innerHTML]="contribution | contributionFormat: true"
+            [routerLink]="['/records', 'persons', 'detail', contribution.agent.pid]">
+          </a>
+          {{ last ? '' : '; ' }}
+        </li>
+      </ul>
+    </ng-container>
+  </div>
+  <div class="col-2"> {{ item.call_number }}</div>
+  <div class="col-2">{{ item.loan.transaction_date | dateTranslate :'medium' }}</div>
+  <!-- NEXT-ROWS :: request detail -->
+  <div class="col-12 mt-2" *ngIf="!isCollapsed">
```

Comment:
col-12 can be replace by col

Reply:
`col` haven't a fixed width. It will adapt to the row to fill this row. With my 'col-12' I force to use the entire row as with ; this will cause a break line and a full width for the second row.